ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -270,61 +269,99 @@ export const TimeChart = forwardRef<ChartInstance, TimeChartProps>(function Time
     __experimentalEChartsOptionsOverride,
     noDataVariant,
     timeZone,
-    tooltipPinnedCoords,
     isStackedBar,
+    enablePinning,
+    pinnedCrosshair,
   ]);
 
+  // Update adjacent charts so tooltip is unpinned when current chart is clicked.
+  useEffect(() => {
+    // Only allow pinning one tooltip at a time, subsequent tooltip click unpins previous.
+    // Multiple tooltips can only be pinned if Ctrl or Cmd key is pressed while clicking.
+    if (tooltipPinnedCoords !== null && lastTooltipPinnedCoords !== null) {
+      if (!isEqual(lastTooltipPinnedCoords, tooltipPinnedCoords)) {
+        setTooltipPinnedCoords(null);
+        if (tooltipPinnedCoords !== null && pinnedCrosshair !== null) {
+          setPinnedCrosshair(null);
+        }
+      }
+    }
+    // tooltipPinnedCoords CANNOT be in dep array or tooltip pinning breaks in the current chart's onClick
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [lastTooltipPinnedCoords, seriesMapping]);
+
   return (
     <Box
       sx={{ height }}
+      // onContextMenu={(e) => {
+      //   // TODO: confirm tooltip pinning works correctly on Windows, should e.preventDefault() be added here
```

Comment:
Left this commented code intentionally in case we do need to add it back but I'd like to avoid calling ` e.preventDefault()` here if possible (`Cmd + Click` is working for me) since it could interfere with default browser behavior once the user pins the tooltip and moves inside it to copy values (or take other actions in the future)